.gitignore

```diff
@@ -10,3 +10,5 @@ src/device/stm32/*.s
 src/device/sam/*.go
 src/device/sam/*.s
 vendor
+llvm
```

Comment:
These exclusions should not be part of this commit/PR.

Reply:
Yeah, I was short cutting a bit. I'll pull them.